test_data/yam/agama_tumbleweed.yaml

```diff
@@ -0,0 +1,7 @@
+---
+os_release_name: openSUSE Tumbleweed
+guided_partitioning:
+  disks:
+    - vdb
+unused_disks:
+  - vda
```

Comment:
we are not doing anything related with the partition, a single file would suffice.